tests/desktop/test_search.py

```diff
@@ -218,6 +216,10 @@ def test_that_search_results_return_20_results_per_page(self, mozwebqa):
     def test_searching_for_addon_type_returns_results_of_correct_type(
         self, mozwebqa, addon_type, term, breadcrumb_component
     ):
+
+        if '-dev' not in mozwebqa.base_url:
+            pytest.xfail('Bug 1117739 - [stage] "Complete themes" search is not returning any results')
```

Comment:
Help me understand what's happening here -- the bug you cite is for stage:
* However the `xfail` will be applied if the test is executed against `stage` and `prod`. Test on production currently pass so I don't see a need for this xfail at present.
* The bug only presents itself when a test for `Complete Themes` are searched against, yet the `xfail`will be applied for all tests in in the `@pytest.mark.parametrize`. Was this intended?